### ISSUE TITLE
Download current version of the list of music (JSON, CSV, etc.)

### ISSUE THREAD

**Issue by liledix4** [liledix4/music#29]
This is not recommended (use online JSON instead for up-to-date information), BUT if there's a need for it (for some reason; possibly for editing, later extraction of data or preservation), then the current version of the list of music can be downloaded and used offline.

## 0.1

- [ ] JSON *(basically, direct download from the source)*

## 0.2 or later

- *Conversion of JSON*
    - [ ] CSV
    - [ ] XML

## 0.3 or later

0.3 could feature **customization**: the possibility of choosing only necessary pieces of data instead of everything at once.

- [ ] JSON with customization
- [ ] CSV with customization
- [ ] XML with customization

Since this version, all later formats are supposed to have the customization feature too.

## 0.4 or later

I'm highly unsure if it's possible to meddle with Excel format just with the client-side JavaScript. But most certainly this could be done on the server side (using PHP).

- [ ] Excel
- [ ] LibreOffice Calc

## 1.0 or later

For extreme needs:

- [ ] SQL
    - Basically, the file with SQL commands will be created.
    - Obviously, it'll be possible to type desired table name.
    - There could be a special option for including the commands for wiping previously created table – in case if it's needed to update the table contents "from scratch".
